css-canvas-medical/styles.css

```diff
@@ -0,0 +1,276 @@
+* {
+  box-sizing: border-box;
+}
+
+section {
+  margin: 0;
+}
+
+body {
+  margin-top: 30px;
+}
+
+.row {
+  display: flex;
+}
+
+.flex-border {
+  border: 2px solid red;
+}
+
+.flex-border > * {
+  outline: 2px dashed chartreuse;
+}
```

Comment:
I'm assuming that these were for you to see the boxes for layout but you should remove unused rulesets in CSS. Having unused CSS created "site bloat" which slows down speed and performance. Minor issue here but in a larger scale project if these are missed they add up.